worker/uniter/runner/jujuc/opened-ports.go

```diff
@@ -45,6 +45,13 @@ augmented with a comma-delimited list of endpoints that the port range
 applies to (e.g. "80/tcp (endpoint1, endpoint2)"). If a port range applies to
 all endpoints, this will be indicated by the presence of a '*' character
 (e.g. "80/tcp (*)").
+
+Opening ports is transactional (i.e. will take place on successfully exiting
+the current hook), and therefore opened-ports will not return any values for
+pending open-port operations run from within the same hook.
+`,
+		Examples: `
+    opened-ports
 `,
```

Comment:
Looking at the handcrafted doc (https://juju.is/docs/juju/hook-tool#opened-ports ), I see the detail you've left out the detail after the example. Was that on purpose? (It started with `Prior to Juju 2.9, ...`, so I'm guessing you thought it wasn't relevant anymore, but wanted to double-check.)

Reply:
Yeah, I don't think we need to explain what happened before Juju 2.9